benchmarks/multiplatform/gradle.properties

```diff
@@ -1,4 +1,4 @@
-compose.version=1.7.1
```

Comment:
We shouldn't merge until we build compose with https://github.com/JetBrains/skiko/pull/1016.

The PR only works with custom Skiko right now:

```
// in skiko
checkout igor.demin/DirectXOffscreenContext
./gradlew publishToMavenLocal -deploy.version=722.0.0

// in benchmarks in build.gradle
    val desktopMain by getting {  
            dependencies {
                ...
                implementation("org.jetbrains.skiko:skiko:722.0.0-SNAPSHOT")  
                implementation("org.jetbrains.skiko:skiko-awt-runtime-macos-arm64:722.0.0-SNAPSHOT")  
```

Reply:
Maybe it is better to make the dependency  reflective for now?

However this way, we cannot compare with the old version anyway. I would add an option to choose between null/GPU/software rendering in addition, with backward compatible defaults. 

I would probably also wait GPU renderers for non-Windows desktop targets to have common code here.